activities/MedicalHistoryLORIS/MedicalHistoryLORIS_schema

```diff
@@ -1,11 +1,11 @@
 {
     "@context": [ "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/contexts/generic",
-        "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/MedicalHistory-LORIS/medicalhistory_context"
+        "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/MedicalHistoryLORIS/MedicalHistoryLORIS_context"
     ],
     "@type": "reproschema:Activity",
-    "@id": "medicalhistory_schema",
+    "@id": "MedicalHistoryLORIS_schema",
```

Comment:
please remove `_schema` at the end of '@id' string i.e. MedicalHistoryLORIS (same as directory name). 

Please make the corrections for '@id' for others too.

Reply:
@sanuann shouldn't the @id match the IRI of the schema?

Reply:
if i remove `_schema`, i believe the `@id` would expand out to `https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/MedicalHistory-LORIS/MedicalHistoryLORIS` which wouldn't be an existing IRI

Reply:
ok makes sense.